<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uVisionClass</class>
 <widget class="QMainWindow" name="uVisionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uVision</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>串口设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>运动参数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>串口标志</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>通讯状态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>按键设定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>模型修正</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>电机状态设定</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>120</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>拍照</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>聚焦</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>细分：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>步数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>280</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>电机状态</string>
    </property>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>更新状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>运动状态：静止</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>到位情况</string>
     </property>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X左到位</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y左到位</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z左到位</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X右到位</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y右到位</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z右到位</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>59</y>
       <width>81</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>轴向锁定状态</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X free</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z free</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y free</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="actClose"/>
    <addaction name="actSave"/>
    <addaction name="actASave"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>设备(&amp;E)</string>
    </property>
    <addaction name="actReset"/>
    <addaction name="actOff"/>
    <addaction name="actOn"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视频(&amp;V)</string>
    </property>
    <addaction name="actPlay"/>
    <addaction name="separator"/>
    <addaction name="actParameter"/>
   </widget>
   <widget class="QMenu" name="menu_I">
    <property name="title">
     <string>图像处理(I)</string>
    </property>
    <addaction name="actOpenImage"/>
    <addaction name="actInstitch"/>
    <addaction name="actAsave"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_V"/>
   <addaction name="menu_I"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actNew">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actASave">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actReset">
   <property name="text">
    <string>复位</string>
   </property>
  </action>
  <action name="actOff">
   <property name="text">
    <string>断开相机</string>
   </property>
  </action>
  <action name="actOn">
   <property name="text">
    <string>启动相机</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actPlay">
   <property name="text">
    <string>视频采集</string>
   </property>
  </action>
  <action name="actStop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="actTakePhoto">
   <property name="text">
    <string>拍照</string>
   </property>
  </action>
  <action name="actParameter">
   <property name="text">
    <string>视频属性</string>
   </property>
  </action>
  <action name="actOpenImage">
   <property name="text">
    <string>打开图像</string>
   </property>
  </action>
  <action name="actEnhance">
   <property name="text">
    <string>图像锐化</string>
   </property>
  </action>
  <action name="actResultAsave">
   <property name="text">
    <string>结果另存</string>
   </property>
  </action>
  <action name="actMatch">
   <property name="text">
    <string>模板匹配</string>
   </property>
  </action>
  <action name="actSift">
   <property name="text">
    <string>SIFT</string>
   </property>
  </action>
  <action name="actInstitch">
   <property name="text">
    <string>图像拼接</string>
   </property>
  </action>
  <action name="actAsave">
   <property name="text">
    <string>结果另存</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="uvision.qrc"/>
 </resources>
 <connections/>
</ui>
